docs/modules/demos/pages/jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data.adoc

```diff
@@ -136,17 +136,71 @@ You should arrive at your workspace:
 image::jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data/jupyter_hub_workspace.png[]
 
 Now you can double-click on the `notebook` folder on the left, open and run the contained file.
-Click on the double arrow (⏩️) to execute the Python scripts.
+Click on the double arrow (⏩️) to execute the Python scripts (click on the image below to go to the notebook file).
 
-image::jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data/jupyter_hub_run_notebook.png[]
+image::jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data/jupyter_hub_run_notebook.png[link="../../../../stacks/jupyterhub-pyspark-hdfs/notebook.ipynb"]
```

Comment:
The link is not working like this, at least when I generate the docs and try to click it (in the generated documentation the files outside of document files, images, attachments etc. are not present).

```suggestion
image::jupyterhub-pyspark-hdfs-anomaly-detection-taxi-data/jupyter_hub_run_notebook.png[link=https://github.com/stackabletech/demos/blob/main/stacks/jupyterhub-pyspark-hdfs/notebook.ipynb,window=_blank]
```

(Also added the `window=_blank` so it opens in a new window and people don't lose the doc page)

Reply:
Thank you! I wasn't sure how best to do that.